### ISSUE TITLE
You're using an old webpack version

### ISSUE THREAD

**Issue by warpdesign** [s-panferov/awesome-typescript-loader#612]
I'm trying to build a typescript project (no tsx/react, only typescript) using webpack and I am getting this error:

```
Error: It looks like you're using an old webpack version without hooks support.
If you're using awesome-script-loader with React storybooks
consider upgrading @storybook/react to at least version 4.0.0-alpha.3
```

Since I'm not using React this is a bit misleading.

I am using `typescript 3.0.1`, `webpack 2.3.3` and `awesome_typescript_loader 5.2.1`

**Comment by sarink** [comment]
The second statement doesn't apply to you, because you're not using awesome-typescript-loader with React storybooks. However, `Error: It looks like you're using an old webpack version without hooks support.` tells you everything you need to know (hint: upgrade webpack).

**Comment by warpdesign** [comment]
You're right: I shouldn't have care about the React stuff.

Updating to latest webpack 4.x fixes the problem.

**Comment by bbugh** [comment]
What is the minimal version of webpack? We're on 3.11 and getting this error. If this library works only for Webpack 4, maybe you could add that to the readme?

**Comment by sarink** [comment]
As with everything npm, dependency versions are listed in `package.json`. You can work backwards through npm releases to find a version of this lib that supports webpack v3.

**Comment by Suyash2810** [comment]
I am trying to build an angular+node project with webpack 4. Previously it was showing the same errors as discussed @warpdesign

I updated webpack and webpack-cli globally and locally. The project was built properly. I started the npm start script and the project opened on localhost. However now the same errors are being shown in the chrome console.

![Screenshot (317)](https://user-images.githubusercontent.com/42000949/63154530-7413e400-c02e-11e9-937b-34374f324cff.png)

This is the index.hbs file which is being rendered on localhost:3000.

![Screenshot (318)](https://user-images.githubusercontent.com/42000949/63154543-7c6c1f00-c02e-11e9-896b-b67f1da1cdc3.png)

The webpack.config.common.js file's code is below:

![Screenshot (319)](https://user-images.githubusercontent.com/42000949/63154991-60b54880-c02f-11e9-9617-4e0446910587.png)

The webpack.config.dev.js is as below:

![Screenshot (320)](https://user-images.githubusercontent.com/42000949/63155041-7b87bd00-c02f-11e9-8398-96d10c473a06.png)

The angular my-app component is not being loaded properly as can be seen. Any help regarding this would be appreciated.
